luskan/src/main/java/org/opencadc/luskan/QueryRunnerImpl.java

```diff
@@ -87,20 +86,17 @@
     public QueryRunnerImpl() {
     }
 
-    @Override
-    protected DataSource getUploadDataSource()
-            throws Exception {
+    @Override protected DataSource getUploadDataSource() throws Exception {
```

Comment:
Please revert this file to previous state - there were no changes needed but now the annotation is messed up

Reply:
I didn't change those methods so the IDE must've done something odd, although can't say I've seen it make formatting changes like that before. Fixed.